app/services/DateTimeService.scala

```diff
@@ -17,24 +17,29 @@
 package services
 
 import config.FrontendAppConfig
-
 import java.time._
 import javax.inject.Inject
 
 class DateTimeService @Inject()(appConfig: FrontendAppConfig) {
 
-  def getTimeStamp: OffsetDateTime = OffsetDateTime.ofInstant( Instant.now() , ZoneOffset.UTC)
+  val year = 2027
+  val month = 12
+  val day = 20
+  val hour = 12
+  val minute = 30
+
+  def getTimeStamp: OffsetDateTime = OffsetDateTime.ofInstant(Instant.now(), ZoneOffset.UTC)
```

Comment:
Seems it is unused and  can be removed. Please crosscheck and remove accordingly